docs/how/add-custom-ingestion-source.md

```diff
@@ -0,0 +1,30 @@
+# How to use a custom ingestion source without forking Datahub?
+
+Adding a custom ingestion source is the easiest way to extend Datahubs ingestion framework to support source systems
+which are not yet officially supported by Datahub.
+
+## What you need to do
+
+First thing to do is building a custom source like it is described in
+the [metadata-ingestion source guide](../../metadata-ingestion/adding-source.md) in your own project.
+
+### How to use this source?
+
+To be able to use this source you just need to do a few things.
+
+1. Build a python package out of your project including the custom source class.
+2. Install this package in your working environment where you are using the Datahub CLI to ingest metadata.
+
+Now you are able to just reference your ingestion source class as a type in the YAML recipe by using the fully qualified
+package name. For example if your project structure looks like this `<project>/src/my-source/custom_ingestion_source.py`
```

Comment:
I'm thinking it might be simplest if we provide a hello_world example of a custom source for people to look at.